src/components/2021/best-of-penn/BlueComponent.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react'
+import s from 'styled-components'
+import { Col, Row } from 'react-bootstrap'
+import Img from 'gatsby-image'
+
+import {
+  RED,
+  BLUE,
+  TAN,
+  WordWithLine,
+  CardName,
+  CardDescription,
+  CardHours,
+  CardAddress,
+  CardTitle,
+  YourPick,
+  CardWrapper,
+} from './shared'
+
+const Background = s.div`
+  background-color: ${BLUE};
+  width: 100%;
+  padding-bottom: 5rem;
+  margin-bottom: 7rem;
+`
+
+const HorizontalLine = s.hr`
+  background: ${TAN}; 
+  padding-bottom: 8px;
+`
+
+const Card = ({ item: { category, img } }) => (
+  <BlueCardWrapper>
+    <Row>
+      <Col xs={11} sm={11} md={11}>
+        <Img fluid={img.src.childImageSharp.fluid} />
+        <CardTitle color={TAN}>{category}</CardTitle>
+      </Col>
+      <Col xs={1} sm={1} md={1} style={{ marginLeft: '-60px' }}>
+        <YourPick color={TAN} bgColor={RED}>
+          <span>
+            YOUR
+            <br />
+            PICK
+          </span>
+        </YourPick>
+      </Col>
+    </Row>
+  </BlueCardWrapper>
+)
+
+const BlueCardWrapper = s(CardWrapper)`
+  text-align: left;
+  color: ${TAN};
+  @media(max-width: 768px) {
+    text-align: center;
+    padding-left: 5%;
+  }
+`
+
+const CardText = ({ item }) => (
+  <BlueCardWrapper>
+    <Col xs={11} sm={11} md={11}>
+      <CardName>{item.place}</CardName>
+      <CardDescription>{item.blurb}</CardDescription>
+      {item.address && (
+        <CardAddress>
+          <strong>ADDRESS:</strong> {item.address}
+        </CardAddress>
+      )}
+      {item.hours && (
+        <CardHours>
+          <strong>HOURS:</strong> {item.hours}
+        </CardHours>
+      )}
```

Comment:
destructure this above (I'm not going to point out every case of a systematic error when it happens, just one)